package.json

```diff
@@ -93,14 +117,14 @@
     "xo": "^0.34.0"
   },
   "scripts": {
-    "generate-constant-typings": "node script/generate-constant-typings",
-    "generate-expressions": "node script/generate-expressions",
-    "generate-dist": "babel lib --out-dir dist --quiet --retain-lines --plugins babel-plugin-unassert,./script/babel-transform-undebug.js,./script/babel-transform-constants.js; node script/copy-dict.js; prettier dist --loglevel error --w",
-    "generate-size": "rollup -c --silent; gzip-size micromark.min.js; gzip-size --raw micromark.min.js",
+    "generate-constant-typings": "node script/generate-constant-typings.mjs",
+    "generate-expressions": "node script/generate-expressions.mjs",
+    "generate-dist": "BUILD=dist rollup -c --silent; node script/copy-dict.js; prettier dist --loglevel error --w",
+    "generate-size": "BUILD=size rollup -c --silent; gzip-size micromark.min.js; gzip-size --raw micromark.min.js",
     "generate": "npm run generate-constant-typings && npm run generate-expressions && npm run generate-dist && npm run generate-size",
     "format": "remark . -qfo && prettier . -w --loglevel warn && xo --fix",
     "pretest-fuzz": "patch-package --patch-dir script/patches; node script/generate-fixtures.mjs",
-    "test-fuzz": "NODE_OPTIONS=\"-r esm\" timeout 15m jsfuzz test/fuzz.js test/fixtures",
+    "test-fuzz": "NODE_OPTIONS=\"-r esm\" jsfuzz test/fuzz.js test/fixtures",
```

Comment:
could you revert this change? I’ve documented how it can be installed (and this script used) on unix now: https://github.com/micromark/micromark#test